Client/Frontend/Strings.swift

```diff
@@ -88,7 +88,7 @@ extension String {
             public static let UniversalAuthenticationReason = MZLocalizedString(
                 key: "Biometry.Screen.UniversalAuthenticationReason.v113",
```

Comment:
We need to update the string key to be able to export the new string for localisation.
```suggestion
                key: "Biometry.Screen.UniversalAuthenticationReason.v115",
```